app.js

```diff
@@ -0,0 +1,140 @@
+      console.log(time[i], customerThisHours);
+      this.dailyCustomers.push(customerThisHours);
+      var salesThisHours = this.calculateHourlySales(customerThisHours);
+      console.log(salesThisHours);
+      this.dailySales.push(salesThisHours);
+    }
+    console.log(this.dailyCustomers);
+    console.log(this.dailySales);
+  }
+};
+var store3 = {
+  name: 'Seattle Center',
+  minCustomer: 11,
+  maxCustomer: 38,
+  avgCookies: 3.7,
+  dailySales: [], //
+  dailyCustomers: [], //
+  calculateNumberCustomers: function() {
+    return Math.random() * (this.maxCustomer - this.minCustomer) + this.minCustomer;
+  },
+  calculateHourlySales: function(customer) {
+    return customer * this.avgCookies;
+  },
+  calculateDailySale: function() {
+    for(var i = 0; i < time.length; i++) {
+      var customerThisHours = this.calculateNumberCustomers();
+      console.log(time[i], customerThisHours);
+      this.dailyCustomers.push(customerThisHours);
+      var salesThisHours = this.calculateHourlySales(customerThisHours);
+      console.log(salesThisHours);
+      this.dailySales.push(salesThisHours);
+    }
+    console.log(this.dailyCustomers);
+    console.log(this.dailySales);
+  }
+};
+
+var store4 = {
+  name: 'Capital Hill',
+  minCustomer: 20,
+  maxCustomer: 38,
+  avgCookies: 2.3,
+  dailySales: [], //
+  dailyCustomers: [], //
+  calculateNumberCustomers: function() {
+    return Math.random() * (this.maxCustomer - this.minCustomer) + this.minCustomer;
+  },
+  calculateHourlySales: function(customer) {
+    return customer * this.avgCookies;
+  },
+  calculateDailySale: function() {
+    for(var i = 0; i < time.length; i++) {
+      var customerThisHours = this.calculateNumberCustomers();
+      console.log(time[i], customerThisHours);
+      this.dailyCustomers.push(customerThisHours);
+      var salesThisHours = this.calculateHourlySales(customerThisHours);
+      console.log(salesThisHours);
+      this.dailySales.push(salesThisHours);
+    }
+    console.log(this.dailyCustomers);
+    console.log(this.dailySales);
+  }
+};
+
+var store5 = {
+  name: 'Seattle Center',
+  minCustomer: 2,
+  maxCustomer: 16,
+  avgCookies: 4.6,
+  dailySales: [], //
+  dailyCustomers: [], //
+  calculateNumberCustomers: function() {
+    return Math.random() * (this.maxCustomer - this.minCustomer) + this.minCustomer;
+  },
+  calculateHourlySales: function(customer) {
+    return customer * this.avgCookies;
+  },
+  calculateDailySale: function() {
+    for(var i = 0; i < time.length; i++) {
+      var customerThisHours = this.calculateNumberCustomers();
+      console.log(time[i], customerThisHours);
+      this.dailyCustomers.push(customerThisHours);
+      var salesThisHours = this.calculateHourlySales(customerThisHours);
+      console.log(salesThisHours);
+      this.dailySales.push(salesThisHours);
+    }
+    console.log(this.dailyCustomers);
+    console.log(this.dailySales);
+  }
+};
```

Comment:
Ok this is a good start. Now you have to actually run the javascript that would access these arrays of daily Sales and render it on the page as an unordered list.